### PR TITLE
Jenkins: Updates jenkins file to pre clean the k8s cluster

### PR DESCRIPTION
**Pull request by ashishranjan738**
This commit updates the jenkins file to pre clean the k8s cluster before the running the integeration tests.

Signed-off-by: Ashish Ranjan <aranjan@redhat.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](https://github.com/rook/rook/blob/master/CONTRIBUTING.md#comments)
- [ ] Add the flag for skipping the CI if this PR does not require a build. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for more details.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -170,6 +170,7 @@ def RunIntegrationTest(k, v) {
         script{
             try{
                 withEnv(["KUBE_VERSION=${v}"]){
+                    sh "tests/scripts/kubeadm.sh clean"
```
Comment (travisn) [inline Jenkinsfile]: nit: how about moving this down a couple lines so we can see the intended version of k8s in the log first

Comment (travisn) [reply Jenkinsfile]: If k8s isn't installed yet (already clean), is this a no-op?

Comment (ashishranjan738) [reply Jenkinsfile]: done. Nope clean doesn't have any effect even if there is no k8s cluster present

**Comment by ashishranjan738** [comment]
@travisn rebased the PR.

File: Jenkinsfile [hunk 2]
```diff
@@ -173,6 +173,7 @@ def RunIntegrationTest(k, v) {
                     unstash 'repo-amd64'
                     echo "running tests on k8s version ${v}"
                     sh 'tests/scripts/makeTestImages.sh load amd64'
+                    sh "tests/scripts/kubeadm.sh clean"
```
Comment (travisn) [inline Jenkinsfile]: do we also need cleanup for the localPathPV.sh? The latest build is failing on the local path setup.

Comment (ashishranjan738) [reply Jenkinsfile]: I think the issue is something else. The image was not even present there. Reorganized the commands.Let's see how the tests go.

**Pull request merged**